catroid/config/lint-baseline.xml

```diff
@@ -23,90 +23,35 @@
   -->
 <issues format="5" by="lint 3.5.2" client="gradle" variant="catroidDebug" version="3.5.2">
 
+    <issue
+        id="MissingPermission"
+        message="Missing permissions required by intent Intent.ACTION_CLOSE_SYSTEM_DIALOGS: android.permission.BROADCAST_CLOSE_SYSTEM_DIALOGS"
+        errorLine1="   sendBroadcast(it);"
+        errorLine2="   ~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/java/org/catrobat/catroid/utils/notifications/StatusBarNotificationManager.java"
+            line="265"
+            column="4"/>
+    </issue>
+
```

Comment:
Could you please delete these lines and test if it still works?

Reply:
sure